src/components/Capture/index.js

```diff
@@ -222,8 +222,3 @@ const CaptureScreen = ({method, useCapture, hasCaptured, ...other})=> (
     }
   </div>
 )
-
```

Comment:
then do you think the actually used (wrapped) components should be the ones being exported instead? As it stands now this component does not have any defaults and it will crash if the props are not set. it might be better not to export it at all, and just export the wrappers instead. This is a minor issue, so do as you see best.